community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -152,7 +152,7 @@
     @Description( "Internal log contexts that should output debug level logging" )
     @Internal
     public static final Setting<List<String>> store_internal_debug_contexts = setting( "store.internal_log.debug_contexts",
-            list( ",", STRING ), "" );
+            list( ",", STRING ), "org.neo4j.diagnostics,org.neo4j.cluster.protocol,org.neo4j.kernel.ha" );
```

Comment:
Given it's likely that users who set `store.internal_log.debug_contexts=` in their own config (albeit at our direction, since it's an undocumented property), they'll likely not include these. Which means they'll inadvertently disable debug for these.

Wouldn't it be better to change the logging statements of interest to INFO level? If we want them all the time, they probably should be anyway, right?